wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/MessageActions/CompleteReactionPicker/ReactionsCollectionView.swift

```diff
@@ -28,18 +28,36 @@ final class ReactionsCollectionView: UICollectionView {
         showsHorizontalScrollIndicator = false
         EmojiCollectionViewCell.register(in: self)
         setupLayout()
+        configureObservers()
         contentInset = UIEdgeInsets(top: 10, left: 30, bottom: 10, right: 30)
     }
 
+    @available(*, unavailable)
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+}
+
+private extension ReactionsCollectionView {
+
     func setupLayout() {
         layout.scrollDirection = .vertical
         layout.minimumLineSpacing = 0
         layout.minimumInteritemSpacing = 0
     }
 
-    @available(*, unavailable)
-    required init?(coder aDecoder: NSCoder) {
-        fatalError("init(coder:) has not been implemented")
+    func configureObservers() {
+        NotificationCenter.default.addObserver(self, selector: #selector(handleKeyboardPresentation), name: UIResponder.keyboardWillShowNotification, object: nil)
```

Comment:
Did you check `KeyboardBlockObserver` ? Can we reuse it?

Reply:
Yes, I've seen it. But it requires more code and make code more complicated and harder to read